composer.lock

```diff
@@ -0,0 +1,117 @@
+{
+    "_readme": [
+        "This file locks the dependencies of your project to a known state",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
+        "This file is @generated automatically"
+    ],
+    "content-hash": "90dddb39371b0ddc80bb0922ff9ebb02",
+    "packages": [
+        {
+            "name": "erusev/parsedown",
+            "version": "1.8.0-beta-7",
+            "source": {
+                "type": "git",
+                "url": "https://github.com/erusev/parsedown.git",
+                "reference": "fe7a50eceb4a3c867cc9fa9c0aa906b1067d1955"
+            },
+            "dist": {
+                "type": "zip",
+                "url": "https://api.github.com/repos/erusev/parsedown/zipball/fe7a50eceb4a3c867cc9fa9c0aa906b1067d1955",
+                "reference": "fe7a50eceb4a3c867cc9fa9c0aa906b1067d1955",
+                "shasum": ""
+            },
+            "require": {
+                "ext-mbstring": "*",
```

Comment:
parsedown itself seems to depend on mbstring some, this won't conflict with our version (until we switch to composer mbstring)?

Reply:
The dproofreaders code already [requires the mbstring extension](https://github.com/DistributedProofreaders/dproofreaders/blob/master/SETUP/INSTALL.md#php) which is what this line specifies in the composer file, so we're good here.